cmd/ndm_daemonset/controller/probe.go

```diff
@@ -27,8 +27,9 @@ import (
 
 // EventMessage struct contains attribute of event message info.
 type EventMessage struct {
-	Action  string                     // Action is event action like attach/detach
-	Devices []*blockdevice.BlockDevice // list of block device details
+	Action        string                     // Action is event action like attach/detach
+	Devices       []*blockdevice.BlockDevice // list of block device details
+	AllowedProbes []string                   // List of probes (given as probe names) to be run for this event. Optional
```

Comment:
Refer to the probe names in `probe` package. Also we may need to export the probe names as they are currently unexported. [Ref](https://github.com/openebs/node-disk-manager/blob/master/cmd/ndm_daemonset/probe/mountprobe.go#L42)

Reply:
Do we need to export them? The only other place outside the probe package where events are pushed to the event channel is the udevevent package. most of the usage will be internal to the probe package I feel. Wdyt?

Reply:
yep. no need to export. I got it clear now.